apps/src/codebridge/InfoPanel/ForTeachersOnly.tsx

```diff
@@ -6,11 +6,11 @@ import TeacherOnlyMarkdown from '@cdo/apps/templates/instructions/TeacherOnlyMar
 
 const ForTeachersOnly: React.FunctionComponent = () => {
   const {levelProperties} = useCodebridgeContext();
-  const teacherMarkdown = levelProperties.teacherMarkdown;
+  const {teacherMarkdown, predictSettings} = levelProperties;
 
   return (
     <div>
-      <PredictSolution />
+      <PredictSolution predictSettings={predictSettings} />
```

Comment:
fyi @mikeharv I don't think Music Lab is using this component for its predict levels (does Music lab have a for teachers only section?), but an fyi that it exists if you want to integrate it somewhere (it shows the answer for a predict level).

Reply:
Thanks Molly. And no, Music Lab doesn't have a teachers only section.